.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Gitignore file shouldn't be inside idea folder. Move is to the root folder of your repository.